src/main/java/usecases/War.java

```diff
@@ -0,0 +1,167 @@
+package usecases;
+
+import entities.Card;
+import presenters.console.Input;
+import presenters.console.Output;
+
+import java.util.Random;
+import java.util.ArrayList;
+import java.util.List;
+
+public class War extends GameTemplate {
+    public final ArrayList[] playingField = new ArrayList[]{new ArrayList<Card>(), new ArrayList<Card>()};
+    private final Input gameInput;
+    private final Output gameOutput;
+    private final static String[] Hierarchy = new String[]{"A", "2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K"};
```

Comment:
Variable naming convention not adhered to here; constants should be all capitals